### ISSUE TITLE
GTA SA menu map, being stretched.

### ISSUE THREAD

**Issue by r3538987** [gennariarmando/sky-ui#5]
Is there a chance eventually to tweak GTA San Andreas menu map section, so map image there doesn't get stretched?
For example in remaster version it looks accurate. This is noticeable when you pay attention to round objects on map. 
![image](https://github.com/gennariarmando/sky-ui/assets/33725345/23a84b15-878b-4319-a870-8ea798d6b28d)

Asked Silent, and he mentioned that this should be possible, but he would prefer that this mod fixes it, as it's main one which adjusts game's menu.

**Comment by gennariarmando** [comment]
Yes but this won't be done in skyui, check out [this](https://github.com/gennariarmando/menu-map) repo, it'll be hopefully updated soon.

**Comment by r3538987** [comment]
Those upcoming changes wont be in skyui because too much rebuilding here is necessary?
